src/services/messengerSync/utils.js

```diff
@@ -157,7 +157,7 @@ const messengerIsActivatedForSchool = async (data) => {
 		data.school = await getSchoolData(data.schoolId);
 	}
 
-	return data.school && Array.isArray(data.school.features) && data.school.features.includes('messenger');
+	return data.school && Array.isArray(data.school.features) && data.school.features.includes(SCHOOL_FEATURES.MESSENGER);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 122. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5417363)